torchrl/envs/libs/pettingzoo.py

```diff
@@ -464,12 +480,18 @@ def _reset(
         self, tensordict: Optional[TensorDictBase] = None, **kwargs
     ) -> TensorDictBase:
 
+        _reset = tensordict.get("_reset", None)
```

Comment:
This crashes when `tensordict is None`

Reply:
Yeah sorry about that, I can't test petting zoo locally so I'm always moving in the dark...